src/justfile.rs

```diff
@@ -406,6 +411,13 @@ impl<'src> Justfile<'src> {
       }
     }
 
+    let empty_vec: Vec<usize> = Vec::new();
+    for submodule in self.modules.values() {
+      for group in submodule.groups() {
+        groups.push((&empty_vec, 0, group));
```

Comment:
This can be:
```suggestion
        groups.push((&[], 0, group));
```

Reply:
With this line:

```rust
groups.push((&recipe.import_offsets, recipe.name.offset, group));
```

Changed to:

```rust
groups.push((recipe.import_offsets.as_slice(), recipe.name.offset, group));
```

Reply:
The contents of `group` is used for sorting, so to make submodule groups sort correctly, think instead of `0`, use `submodule.name.unwrap().offset`. `unwrap()` is okay here, because any non-root justfile will have `name` set.

Reply:
Updated.